docs/.storybook/preview.tsx

```diff
@@ -1,3 +1,8 @@
+import '@fontsource/roboto/300.css';
+import '@fontsource/roboto/400.css';
+import '@fontsource/roboto/500.css';
+import '@fontsource/merriweather/400.css';
```

Comment:
Is there a new need to add these here now?  The intended way they would get imported was they they were to be imported in the `index.ts` of `@ui-kit-2022/theme` so that when consumers import the the theme (which this file does), they would get loaded.  Lets try to see if we can fix that before we import here.

Reply:
I imported these here since the fonts weren't appearing in Storybook.  Even though stories are wrapped with the theme and theme provider.  

This was the solution @Honglin-Lu and @iandilling-adaptive came up with if I recall correctly.  I'll speak with them about the problem and what they've tried then get back to you.